### PR TITLE
Update NativeLibraryInterop docs

### PR DESCRIPTION
**Pull request by rachelkang**
Small tweaks to update instructions and fix typos

### CODE REVIEW

File: docs/maui/native-library-interop/get-started.md [hunk 1]
```diff
@@ -93,20 +92,30 @@ On the native side, make updates in _template/macios/native/NewBinding/NewBindin
...
 
 Back on the .NET side, we are now ready to interop with the native library:
-Run `dotnet build` from _template/android/NewBinding.Android.Binding_ to test everything is plugged in correctly and good to go. (Note: This step will require that you have JDK 17 installed)
+1. Run `dotnet build` from _template/android/NewBinding.Android.Binding_ to test everything is plugged in correctly and good to go. (Note: This step will require that you have JDK 17 installed)
+1. Reference the Android binding dependency by adding the following code to _template/sample/MauiSample.csproj_. Replace _{yourNewBindingLibrary.aar}_ with your AndroidLibrary .aar file name. (Note: The gradle/maven dependencies often need to be explicitly referenced, as they are not automatically bundled into your library. The _build.gradle.kts_ file is configured to copy depencies into a bin/outputs/deps folder for you to reference in your .NET sample)
+
+```xml
+<ItemGroup Condition="$(TargetFramework.Contains('android'))">
+    <AndroidLibrary Include="..\android\native\newbinding\build\outputs\deps\{yourNewBindingLibrary.aar}">
```
Comment (pjcollins) [inline docs/maui/native-library-interop/get-started.md:113]:
```suggestion
    <AndroidLibrary Include="..\android\native\newbinding\bin\outputs\deps\{yourNewBindingLibrary.aar}">
```
Small update for the latest path here

File: docs/maui/native-library-interop/get-started.md [hunk 2]
```diff
@@ -93,20 +92,30 @@ On the native side, make updates in _template/macios/native/NewBinding/NewBindin
...
-3. Ensure the Android Studio project builds successfully and you are satisfied with the APIs.
+1. Write the API definitions for the native library APIs of interest.
+1. Ensure the Android Studio project builds successfully and you are satisfied with the APIs.
 
 Back on the .NET side, we are now ready to interop with the native library:
-Run `dotnet build` from _template/android/NewBinding.Android.Binding_ to test everything is plugged in correctly and good to go. (Note: This step will require that you have JDK 17 installed)
+1. Run `dotnet build` from _template/android/NewBinding.Android.Binding_ to test everything is plugged in correctly and good to go. (Note: This step will require that you have JDK 17 installed)
+1. Reference the Android binding dependency by adding the following code to _template/sample/MauiSample.csproj_. Replace _{yourNewBindingLibrary.aar}_ with your AndroidLibrary .aar file name. (Note: The gradle/maven dependencies often need to be explicitly referenced, as they are not automatically bundled into your library. The _build.gradle.kts_ file is configured to copy depencies into a bin/outputs/deps folder for you to reference in your .NET sample)
```
Comment (pjcollins) [inline docs/maui/native-library-interop/get-started.md:109]:
```suggestion
1. Reference any Android binding dependencies by adding the following code to _template/sample/MauiSample.csproj_. Replace _{yourDependencyLibrary.aar}_ with any required .aar files from the library you are binding. (Note: The gradle/maven dependencies often need to be explicitly referenced, as they are not automatically bundled into your library. The _build.gradle.kts_ file is configured to copy dependencies into a bin/outputs/deps folder for you to reference in your application)
```
Small update to try to improve clarity that certain .aar files from the library being bound may need to be explicitly included in the application.

File: docs/maui/native-library-interop/get-started.md (hunk 1 above)
Comment (rachelkang) [reply docs/maui/native-library-interop/get-started.md:113]: Hmm... for me, it is showing up in the build folder? I tried again from the template directly in the latest version of the repo
image

Comment (pjcollins) [reply docs/maui/native-library-interop/get-started.md]: Oh sorry I missed an update on the template, there is a ${buildDir} property that can be used over there which should update this path to something like `bin\Release\net8.0-android\outputs\deps`

**Review by pjcollins — APPROVED** [review]

**Review by jfversluis — APPROVED** [review]

**Pull request merged**